<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="4" ignored="0" total="6" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="37786" started-at="2022-09-18T15:04:23Z" finished-at="2022-09-18T15:05:00Z">
    <groups>
      <group name="Purchase">
        <method signature="StandAloneTest.StandAlone(java.util.HashMap)[pri:0, instance:rahulshettyacademy.tests.StandAloneTest@799f10e1]" name="StandAlone" class="rahulshettyacademy.tests.StandAloneTest"/>
      </group> <!-- Purchase -->
      <group name="ErrorHandling">
        <method signature="ErrorValidation.LoginErrorValidation()[pri:0, instance:rahulshettyacademy.tests.ErrorValidation@65fb9ffc]" name="LoginErrorValidation" class="rahulshettyacademy.tests.ErrorValidation"/>
      </group> <!-- ErrorHandling -->
    </groups>
    <test name="Submit Order Test" duration-ms="30113" started-at="2022-09-18T15:04:23Z" finished-at="2022-09-18T15:04:53Z">
      <class name="rahulshettyacademy.tests.StandAloneTest">
        <test-method status="PASS" signature="launchApplication()[pri:0, instance:rahulshettyacademy.tests.StandAloneTest@799f10e1]" name="launchApplication" is-config="true" duration-ms="8968" started-at="2022-09-18T15:04:23Z" finished-at="2022-09-18T15:04:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method status="FAIL" signature="StandAlone(java.util.HashMap)[pri:0, instance:rahulshettyacademy.tests.StandAloneTest@799f10e1]" name="StandAlone" duration-ms="7687" started-at="2022-09-18T15:04:32Z" data-provider="getData" finished-at="2022-09-18T15:04:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Ch@@2022, email=chaimaammar.simplon@gmail.com, productName=ZARA COAT 3}]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.ElementClickInterceptedException">
            <message>
              <![CDATA[element click intercepted: Element is not clickable at point (1047, 651)
  (Session info: chrome=104.0.5112.101)
Build info: version: '4.3.0', revision: 'a4995e2c09*'
System info: host: 'chaimaammar-simplon-gmail-com', ip: '192.168.1.10', os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.0-46-generic', java.version: '1.8.0_342'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [9c4a9576028d5771a5908ebfc5992e8f, clickElement {id=cb2d5498-d3a5-48cb-aa26-0722a4712c98}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 104.0.5112.101, chrome: {chromedriverVersion: 104.0.5112.79 (3cf3e8c8a07d..., userDataDir: /tmp/.com.google.Chrome.FI8eWi}, goog:chromeOptions: {debuggerAddress: localhost:38769}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:cdp: ws://localhost:38769/devtoo..., se:cdpVersion: 104.0.5112.101, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on LINUX (9c4a9576028d5771a5908ebfc5992e8f)] -> css selector: .action__submit]
Session ID: 9c4a9576028d5771a5908ebfc5992e8f]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element is not clickable at point (1047, 651)
  (Session info: chrome=104.0.5112.101)
Build info: version: '4.3.0', revision: 'a4995e2c09*'
System info: host: 'chaimaammar-simplon-gmail-com', ip: '192.168.1.10', os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.0-46-generic', java.version: '1.8.0_342'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [9c4a9576028d5771a5908ebfc5992e8f, clickElement {id=cb2d5498-d3a5-48cb-aa26-0722a4712c98}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 104.0.5112.101, chrome: {chromedriverVersion: 104.0.5112.79 (3cf3e8c8a07d..., userDataDir: /tmp/.com.google.Chrome.FI8eWi}, goog:chromeOptions: {debuggerAddress: localhost:38769}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:cdp: ws://localhost:38769/devtoo..., se:cdpVersion: 104.0.5112.101, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on LINUX (9c4a9576028d5771a5908ebfc5992e8f)] -> css selector: .action__submit]
Session ID: 9c4a9576028d5771a5908ebfc5992e8f
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:569)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:257)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:78)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:52)
at com.sun.proxy.$Proxy15.click(Unknown Source)
at rahulshettyacademy.pageobjects.CheckoutPage.submitOrder(CheckoutPage.java:47)
at rahulshettyacademy.tests.StandAloneTest.StandAlone(StandAloneTest.java:50)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:750)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.ElementClickInterceptedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StandAlone -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:rahulshettyacademy.tests.StandAloneTest@799f10e1]" name="tearDown" is-config="true" duration-ms="246" started-at="2022-09-18T15:04:40Z" finished-at="2022-09-18T15:04:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="launchApplication()[pri:0, instance:rahulshettyacademy.tests.StandAloneTest@799f10e1]" name="launchApplication" is-config="true" duration-ms="6279" started-at="2022-09-18T15:04:41Z" finished-at="2022-09-18T15:04:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method status="FAIL" signature="StandAlone(java.util.HashMap)[pri:0, instance:rahulshettyacademy.tests.StandAloneTest@799f10e1]" name="StandAlone" duration-ms="5386" started-at="2022-09-18T15:04:47Z" data-provider="getData" finished-at="2022-09-18T15:04:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Ch@@2022, email=chaimaammar.simplon@gmail.com, productName=ADIDAS ORIGINAL}]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.ElementClickInterceptedException">
            <message>
              <![CDATA[element click intercepted: Element is not clickable at point (1047, 653)
  (Session info: chrome=104.0.5112.101)
Build info: version: '4.3.0', revision: 'a4995e2c09*'
System info: host: 'chaimaammar-simplon-gmail-com', ip: '192.168.1.10', os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.0-46-generic', java.version: '1.8.0_342'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [7f4f04c13808cb18d8116fd4337e0b67, clickElement {id=783e7c23-8c7a-4a26-9cf8-627e69ed015e}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 104.0.5112.101, chrome: {chromedriverVersion: 104.0.5112.79 (3cf3e8c8a07d..., userDataDir: /tmp/.com.google.Chrome.o4PkrH}, goog:chromeOptions: {debuggerAddress: localhost:45633}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:cdp: ws://localhost:45633/devtoo..., se:cdpVersion: 104.0.5112.101, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on LINUX (7f4f04c13808cb18d8116fd4337e0b67)] -> css selector: .action__submit]
Session ID: 7f4f04c13808cb18d8116fd4337e0b67]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element is not clickable at point (1047, 653)
  (Session info: chrome=104.0.5112.101)
Build info: version: '4.3.0', revision: 'a4995e2c09*'
System info: host: 'chaimaammar-simplon-gmail-com', ip: '192.168.1.10', os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.0-46-generic', java.version: '1.8.0_342'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [7f4f04c13808cb18d8116fd4337e0b67, clickElement {id=783e7c23-8c7a-4a26-9cf8-627e69ed015e}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 104.0.5112.101, chrome: {chromedriverVersion: 104.0.5112.79 (3cf3e8c8a07d..., userDataDir: /tmp/.com.google.Chrome.o4PkrH}, goog:chromeOptions: {debuggerAddress: localhost:45633}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:cdp: ws://localhost:45633/devtoo..., se:cdpVersion: 104.0.5112.101, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on LINUX (7f4f04c13808cb18d8116fd4337e0b67)] -> css selector: .action__submit]
Session ID: 7f4f04c13808cb18d8116fd4337e0b67
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:569)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:257)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:78)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:52)
at com.sun.proxy.$Proxy15.click(Unknown Source)
at rahulshettyacademy.pageobjects.CheckoutPage.submitOrder(CheckoutPage.java:47)
at rahulshettyacademy.tests.StandAloneTest.StandAlone(StandAloneTest.java:50)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:750)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.ElementClickInterceptedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StandAlone -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:rahulshettyacademy.tests.StandAloneTest@799f10e1]" name="tearDown" is-config="true" duration-ms="234" started-at="2022-09-18T15:04:52Z" finished-at="2022-09-18T15:04:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="OrderHistoryTest()[pri:0, instance:rahulshettyacademy.tests.StandAloneTest@799f10e1]" name="OrderHistoryTest" duration-ms="0" started-at="2022-09-18T15:04:53Z" depends-on-methods="rahulshettyacademy.tests.StandAloneTest.StandAlone" finished-at="2022-09-18T15:04:53Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method StandAloneTest.OrderHistoryTest()[pri:0, instance:rahulshettyacademy.tests.StandAloneTest@799f10e1] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method StandAloneTest.OrderHistoryTest()[pri:0, instance:rahulshettyacademy.tests.StandAloneTest@799f10e1] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:750)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistoryTest -->
      </class> <!-- rahulshettyacademy.tests.StandAloneTest -->
    </test> <!-- Submit Order Test -->
    <test name="Error Validation" duration-ms="37786" started-at="2022-09-18T15:04:23Z" finished-at="2022-09-18T15:05:00Z">
      <class name="rahulshettyacademy.tests.ErrorValidation">
        <test-method status="PASS" signature="launchApplication()[pri:0, instance:rahulshettyacademy.tests.ErrorValidation@65fb9ffc]" name="launchApplication" is-config="true" duration-ms="8194" started-at="2022-09-18T15:04:23Z" finished-at="2022-09-18T15:04:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method status="SKIP" signature="LoginErrorValidation()[pri:0, instance:rahulshettyacademy.tests.ErrorValidation@65fb9ffc]" name="LoginErrorValidation" duration-ms="3267" started-at="2022-09-18T15:04:31Z" finished-at="2022-09-18T15:04:34Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email  password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email  password.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at rahulshettyacademy.tests.ErrorValidation.LoginErrorValidation(ErrorValidation.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:750)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:rahulshettyacademy.tests.ErrorValidation@65fb9ffc]" name="tearDown" is-config="true" duration-ms="479" started-at="2022-09-18T15:04:34Z" finished-at="2022-09-18T15:04:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="launchApplication()[pri:0, instance:rahulshettyacademy.tests.ErrorValidation@65fb9ffc]" name="launchApplication" is-config="true" duration-ms="6623" started-at="2022-09-18T15:04:35Z" finished-at="2022-09-18T15:04:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method status="FAIL" signature="LoginErrorValidation()[pri:0, instance:rahulshettyacademy.tests.ErrorValidation@65fb9ffc]" name="LoginErrorValidation" duration-ms="1037" started-at="2022-09-18T15:04:41Z" finished-at="2022-09-18T15:04:42Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email  password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email  password.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at rahulshettyacademy.tests.ErrorValidation.LoginErrorValidation(ErrorValidation.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:750)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:rahulshettyacademy.tests.ErrorValidation@65fb9ffc]" name="tearDown" is-config="true" duration-ms="334" started-at="2022-09-18T15:04:43Z" finished-at="2022-09-18T15:04:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="launchApplication()[pri:0, instance:rahulshettyacademy.tests.ErrorValidation@65fb9ffc]" name="launchApplication" is-config="true" duration-ms="6058" started-at="2022-09-18T15:04:43Z" finished-at="2022-09-18T15:04:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method status="FAIL" signature="ProductErrorValidation()[pri:0, instance:rahulshettyacademy.tests.ErrorValidation@65fb9ffc]" name="ProductErrorValidation" duration-ms="10678" started-at="2022-09-18T15:04:49Z" finished-at="2022-09-18T15:05:00Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Expected condition failed: waiting for visibility of element located by By.cssSelector: .mb-3 (tried for 5 second(s) with 500 milliseconds interval)
Build info: version: '4.3.0', revision: 'a4995e2c09*'
System info: host: 'chaimaammar-simplon-gmail-com', ip: '192.168.1.10', os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.0-46-generic', java.version: '1.8.0_342'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 104.0.5112.101, chrome: {chromedriverVersion: 104.0.5112.79 (3cf3e8c8a07d..., userDataDir: /tmp/.com.google.Chrome.hrlxCR}, goog:chromeOptions: {debuggerAddress: localhost:45835}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:cdp: ws://localhost:45835/devtoo..., se:cdpVersion: 104.0.5112.101, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 1504203b4eba5a463522da61d4ee7816]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of element located by By.cssSelector: .mb-3 (tried for 5 second(s) with 500 milliseconds interval)
Build info: version: '4.3.0', revision: 'a4995e2c09*'
System info: host: 'chaimaammar-simplon-gmail-com', ip: '192.168.1.10', os.name: 'Linux', os.arch: 'amd64', os.version: '5.15.0-46-generic', java.version: '1.8.0_342'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 104.0.5112.101, chrome: {chromedriverVersion: 104.0.5112.79 (3cf3e8c8a07d..., userDataDir: /tmp/.com.google.Chrome.hrlxCR}, goog:chromeOptions: {debuggerAddress: localhost:45835}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:cdp: ws://localhost:45835/devtoo..., se:cdpVersion: 104.0.5112.101, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 1504203b4eba5a463522da61d4ee7816
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:87)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:231)
at rahulshettyacademy.AbstractComponents.AbstractComponents.waitForElementToAppear(AbstractComponents.java:36)
at rahulshettyacademy.pageobjects.ProductCatalogue.getProductList(ProductCatalogue.java:40)
at rahulshettyacademy.tests.ErrorValidation.ProductErrorValidation(ErrorValidation.java:50)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:750)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:rahulshettyacademy.tests.ErrorValidation@65fb9ffc]" name="tearDown" is-config="true" duration-ms="141" started-at="2022-09-18T15:05:00Z" finished-at="2022-09-18T15:05:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- rahulshettyacademy.tests.ErrorValidation -->
    </test> <!-- Error Validation -->
  </suite> <!-- Suite -->
</testng-results>
